x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/index.tsx

```diff
@@ -22,17 +24,28 @@ import { AllRulesTabs, RulesTableToolbar } from './rules_table_toolbar';
  *   * Import/Export
  */
 export const AllRules = React.memo(() => {
-  const [activeTab, setActiveTab] = useState(AllRulesTabs.rules);
-
   useInitializeRulesTableSavedState();
   useSyncRulesTableSavedState();
 
   return (
     <>
       <RulesManagementTour />
-      <RulesTableToolbar activeTab={activeTab} onTabChange={setActiveTab} />
-      <EuiSpacer />
-      <RulesTables selectedTab={activeTab} />
+      <Switch>
+        <Route path="/rules/monitoring">
+          <>
+            <RulesTableToolbar />
+            <EuiSpacer />
+            <RulesTables selectedTab={AllRulesTabs.monitoring} />
+          </>
+        </Route>
+        <Route path="/rules">
+          <>
+            <RulesTableToolbar />
+            <EuiSpacer />
+            <RulesTables selectedTab={AllRulesTabs.rules} />
+          </>
+        </Route>
+      </Switch>
```

Comment:
Should we add a default redirect to the rules management tab if nothing else matches? Not sure about that, though🤔

Reply:
I've added a redirect `rules` -> `rules/management` to handle compatibility issues and avoid refactoring the way how existing routing handles the rules page.